misc/prototypes.h

```diff
@@ -371,9 +371,10 @@ enum uft_socket_domain {
 enum uft_socket_type {
 	SOCK_STREAM = 1, SOCK_DGRAM, SOCK_RAW, SOCK_RDM, SOCK_SEQPACKET, SOCK_DCCP,
 	SOCK_PACKET = 10,
+	SOCK_NONBLOCK = 04000, SOCK_CLOEXEC = 02000000,
 };
 enum uft_socket_flag {
-	 SOCK_NONBLOCK = 04000, SOCK_CLOEXEC = 02000000,
+	SOCK_NONBLOCK = 04000, SOCK_CLOEXEC = 02000000,
```

Comment:
I'm curious why they are in octal notation, can we just have them in hex?

Reply:
There are written in octal notation in `/usr/include/x86_64-linux-gnu/bits/socket_type.h`. It's not a problem to convert to hex, but I think it looks fine to keep same as where it was written.

Reply:
Ok, that makes sense.